src/mlstacks/terraform/aws-remote-state/main.tf

```diff
@@ -0,0 +1,49 @@
+provider "aws" {
```

Comment:
And also with Payer config: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_request_payment_configuration